src/dbwrapper.cpp

```diff
@@ -42,13 +42,16 @@ wxSharedPtr<wxSQLite3Database> mmDBWrapper::Open(const wxString &dbpath, const w
 
     int err = SQLITE_OK;
     wxString errStr=wxEmptyString;
+    wxSQLite3CipherSQLCipher cipher;
+    cipher.InitializeVersionDefault(4);
```

Comment:
What is the reason for using version 4 here? The API documentation for wxSQLite3Cipher suggests 3 is the max.

https://utelle.github.io/wxsqlite3/docs/html/classwx_s_q_lite3_cipher_s_q_l_cipher.html#a8df5f826a6582c238ff83de491a852a2

> Currently version 1, 2 and 3 of SQLCipher are 
supported. If the version number is invalid, version 3 is assumed.

Reply:
It seems the documentation is out of date. Looking at the code I see 4 is now the default:

https://github.com/utelle/wxsqlite3/blob/1e387a4bea65ada2fdb1b7c95621dc84afb48cb7/src/wxsqlite3.cpp#L6257C1-L6269C13